src/components/standalone/users_objects/CreateOrEditHostSetDrawer.vue

```diff
@@ -133,6 +155,36 @@ function runFieldValidators(
   return validators.every((validator) => validator.valid)
 }
 
+async function getMatchedItemsName(matches: string[]): Promise<string> {
+  try {
+    const res = await ubusCall('ns.objects', 'get-info', { ids: matches })
+    const names: string[] = []
+    for (const match of Object.values(res.data.info) as MatchInfo[]) {
+      if (match.type === 'redirect') {
```

Comment:
```suggestion
    const res: MatchInfoResponse = await ubusCall('ns.objects', 'get-info', { ids: matches })
    const names: string[] = []
    for (const match of res.data.info) {
      if (match.type == 'redirect') {
```

Reply:
I got 
`
The error "TypeError: res.data.info is not iterable" indicates that res.data.info is not an array or another iterable object. This can happen if the structure of the response data is not as expected.`

I needed to adapt